src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -72,49 +66,54 @@ A detailed list of the WebExtension APIs supported in Firefox for Android is pro
+- [`sidebarAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sidebarAction) and the related [`sidebar_action`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/sidebar_action) `manifest.json` key, due to the limited screen real estate on Android devices sidebars, such as the browser history, are presented in full browser tabs. Where possible, you should move any sidebar content to tabs.
+- [`windows`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/windows), as there is only one Firefox on Android 'window', and the browser cannot open or otherwise manipulate additional browser windows.
 
-Also, in both [`pageAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction) and [`browserAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction) popup content is opened in a new tab and persists until the user manually closes the tab.
-
-You can also manipulate tabs on Firefox for Android. The [`tabs`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs) API enables you to perform most of the actions you can on the desktop, the main exceptions are:
-
-- zoom features, Firefox for Android has one zoom level only, which the user can override with a pinch gesture on the page.
-- features related to selecting and moving tabs, again as these features are not supported on Android.
-- the ability to detect a tab's language or muted status.
 
 ### Other UI related API and `manifest.json` key differences
 
 There are some other UI features not supported, these are:
 
 - [`bookmarks`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks), which means you cannot manipulate the user's bookmarks, although the user can do this themselves through the UI.
-- [`browsingData`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData), which means you cannot offer users features to clear browser data such as history, downloads, passwords, and alike.
 - The [`chrome_url_overrides`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/chrome_url_overrides) and [`chrome_settings_overrides`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/chrome_settings_overrides) `manifest.json` keys, which means you cannot add custom home and new tab pages.
-- [`contextMenus`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/contextMenus), which means you cannot add options to context menus.
+- [`menus`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/menus), which means you cannot add options to context menus.
```

Comment:
**issue**: I think that in this context here we may want to keep also the previous mention to `contextMenus`, even if it is ok to also add a mention and link to the `menus` API given that we are updating this point.

<details>

As some additional "context" that may help understanding the two API namespaces `menus` and `contextMenus`
-  `contextMenus` and `menus` API namespaces are technically an alias of each other in many ways
- `menus` API namespace is currently only available in Firefox, the other browsers do not support it at all
- `contextMenus` is the API namespace is the one we share with the other vendors 
- in `menus` there is a subset of features supported on desktop that are only provided from the `menus` API namespace and not by `contextMenus`

None of this is relevant in these android-related doc pages, but I though to mention it to you in case it helps to clear potential confusions about the two API namespaces. 

</details>

Reply:
The main reason for making this change is that if you follow the old link it lands you on a description of the `menu` API. In addition, we don't include `contextMenus` in the API documentation.